### PR TITLE
Sharath Balaji T week 1 assignment 

### PR DESCRIPTION
**Pull request by sharath-2003**
img

### CODE REVIEW

File: week_1/html1.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head class="hdd1">
+<title>My Guest Book</title>
+<link rel="stylesheet" type="text/css" href="css1.css">
+<link href="https://fonts.googleapis.com/css2?family=Kalam&display=swap" rel="stylesheet">
+<div class="tagr"><h1 class="hd">My Guest Book</h1>
+<h3 class="hd1">Please sign My Guest Book.Thanks!!!</h3></div>
```
Comment (hrushikeshj) [inline week_1/html1.html:8]: Add this inside the `body`.

**Review by hrushikeshj — APPROVED** [review]
Hey, good job with the styling and formatting!
It is good practice to [format HTML](https://code.visualstudio.com/docs/languages/html#_formatting) code, it makes it more readable.

File: week_1/html1.html (hunk 1 above)
Comment (sharath-2003) [reply week_1/html1.html:8]: OK

Comment (sharath-2003) [reply week_1/html1.html]: The changes are made, please  verify it.

Comment (hrushikeshj) [reply week_1/html1.html]: Looks good

**Pull request closed without merging**